static/src/js/util/collections.js

```diff
@@ -169,7 +169,13 @@ export const buildCollectionSearchParams = (params) => {
   }
 
   // If there is a keyword, add the wildcard character between words and following the final character
-  const keywordWithWildcard = !keyword ? undefined : `${keyword.replace(/\s+/g, '* ')}*`
+  // If the user types a quoted string, don't add the wildcard character
+  let keywordWithWildcard
+  if (keyword && keyword.match(/(".*")/)) {
+    keywordWithWildcard = keyword
+  } else if (keyword) {
+    keywordWithWildcard = `${keyword.replace(/\s+/g, '* ')}*`
+  }
```

Comment:
The only comment I have, which I have no solution for, is when they search for "word" and word, they might not get the same result, because of the way you're adding the wildcards(there must be a requirement). If you use the CMR endpoint directly, they should return the same result.